packages/builders/src/components/ActionRow.ts

```diff
@@ -34,7 +34,7 @@ export class ActionRow<T extends ActionRowComponent = ActionRowComponent> implem
 	 * Sets the components in this action row
 	 * @param components The components to set this row to
 	 */
-	public setComponents(components: T[]) {
+	public setComponents(...components: T[]) {
```

Comment:
I don't really agree with dropping support for `.set([Button(), Button()])`, if anything both should be supported imo, but I'll let the other @discordjs/the-big-4 give their input

Reply:
just for set..? so is addXs okay as it is?

Reply:
You can just do `.set(...[Button(), Button()])`, the argument spread existed for quite some time to fix this.

Plus, doing `.flat` has quite a performance impact, and can lead to bugs if we supported tuples or arrays within spread arguments.

Reply:
Can’t you just do .set(Button(), Button()) though?

Reply:
It was just an example of an array that's spread, like, defined elsewhere (but inlined for brevity).

Reply:
Yeah but what I’m saying is that you’re not dropping support for anything here, it still works

Reply:
https://github.com/discordjs/discord.js/pull/7395#discussion_r799764489